DSE_6.8_Release_Notes.md

```diff
@@ -4,6 +4,31 @@ Components that are indicated with an asterisk (&ast;) (if any) are known to be
 
 Release notes of versions prior to 6.8.4 can be found [here](https://docs.datastax.com/en/dse/6.8/dse-admin/datastax_enterprise/releaseNotes/RNdse.html).
 
+# Release notes for 6.8.60
+8 September 2025
```

Comment:
The date seems wrong, it shall either be today or yesterday - I am unsure when we made the release.